src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
+export const TitleWrapper = styled.div`
+    display: flex;
+    gap: 12px;
+    align-items: center;
+    justify-self: right;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        gap: 8px;
+        margin-top: 8px;
+        justify-self: left;
+    }
+`;
+
+export const StyledVideoIcon = styled(VideoIcon)`
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        width: 17px;
+        height: 17px;
+    }
+`;
+
+export const TitleText = styled.div`
+    font-family: poppins;
+    font-weight: 500;
+    font-size: 0.8125rem;
+    font-size: clamp(0.8125rem, 0.675rem + 0.6875vw, 1.5rem);
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        font-size: 13px;
+    }
+`;
+
+export const ButtonsWrapper = styled.div`
+    display: flex;
+    gap: 16px;
+    flex-grow: 1;
+    justify-self: left;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        gap: 12px;
+        flex-grow: 0;
+        margin-top: 8px;
+        justify-self: right;
+    }
+`;
+
+export const HeaderButton = styled.button`
+    color: ${({ theme }) => theme.color.white};
+    background: none;
+    border: none;
+    /* border: 1px solid ${({ theme }) => theme.color.white}; */
+    border-radius: 24px;
+    font-size: 14px;
+    font-family: poppins;
+    font-weight: 600;
+    padding: 8px 24px;
+
+    &:hover{
+        cursor: pointer;
+    }
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        font-size: 12px;
+        padding: 8px 12px;
+    }
+`;
+
+export const Form = styled.div`
+    display: flex;
+    align-items: center;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        grid-column-start: 1;
+        grid-column-end: 3;
+        justify-content: center;
+    }
+`;
+
+export const StyledSearchIcon = styled(SearchIcon)`
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        width: 16px;
+    }
+`;
+
+export const SearchBarIcon = styled.div`
+    color: ${({ theme }) => theme.color.waterloo};
+    background-color: ${({ theme }) => theme.color.white};
+    height: 48px;
+    width: 50px;
+    border-radius: 33px 0 0 33px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 44px;
+    }
+`;
+
+export const SearchBar = styled.input`
```

Comment:
Same thing with `font-family`,
I think there should also be some padding

![image](https://user-images.githubusercontent.com/117176251/220742212-d41bbaab-4fe5-440d-9c17-e1399d8113ba.png)
How about that?

Reply:
I decided to add `padding: 19px;` and move SearchBarIcon to the right to make it look more similar as in the project
![obraz](https://user-images.githubusercontent.com/118211320/220761190-2beb2156-2fe9-4d9b-b93d-a8cae2204672.png)

Reply:
thank you for your help I appreciate that